integration-tests/cleaner-test/pom.xml

```diff
@@ -42,10 +42,47 @@
         <version>1.12.0</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.awaitility</groupId>
+        <artifactId>awaitility</artifactId>
+        <version>3.1.6</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
```

Comment:
Do you need to add mockito, junit and hamcrest here in dependencyManagement section? Those libraries are already configured in root pom.xml (root repository folder)